scrapy/core/downloader/contextfactory.py

```diff
@@ -46,7 +46,9 @@ def getCertificateOptions(self):
             #   not calling super(..., self).__init__
             return CertificateOptions(verify=False,
                         method=getattr(self, 'method',
-                                       getattr(self, '_ssl_method', None)))
+                                       getattr(self, '_ssl_method', None)),
+                        fixBrokenPeers=True,
```

Comment:
I have no idea what it means, but in Twisted docs there is a note:

> This option is now off by default, because it causes problems with connections between peers using OpenSSL 0.9.8a.

Is it required to turn it on, or is it turned on just in case? I'm fine with turning it on, but we should keep an eye on it then.